app/components/util/DeFiModal.tsx

```diff
@@ -29,7 +31,16 @@ const DeFiModal = ({
       >
         <ScrollView contentContainerStyle={styles.scrollContainer}>
           <View style={[AppStyles.containerHorizontal]}>
-            <H2 text={title} style={AppStyles.mr20} />
+            {isBeta ? (
+              <>
+                <H2 text={title} />
+                <View style={styles.betaContainer}>
+                  <Text style={styles.beta}> Beta</Text>
+                </View>
+              </>
+            ) : (
+              <H2 text={title} style={AppStyles.mr20} />
+            )}
```

Comment:
Wrapping is not really nice 😉 
![image](https://user-images.githubusercontent.com/85513542/179790246-d1fc7bb2-bd20-4357-b787-c04cf1adde2e.png)

Reply:
<img width="325" alt="image" src="https://user-images.githubusercontent.com/4957647/179811370-53963512-e565-48af-aae7-b04507959b83.png">

adding that `AppStyle` doesn't change anything for me. I don't know how to do an "inline" change for this text in RN. That it is not handled like a separate element.